docs/_sass/_main.scss

```diff
@@ -402,19 +404,10 @@ button {
     font-weight: bold;
     cursor: pointer;
     color: $color-dark;
+    background-color: $color-gray1;
 
-    @include card-border($color-gray2, 1px);
```

Comment:
Why did we just get rid of the card-borders? I really liked them stylistically and couldn't find any context for that decision.